.eslintrc.yaml

```diff
@@ -0,0 +1,18 @@
+env:
+  node: true
+  es6: true
+
+rules:
+  indent: [2, 2]
+  quotes: [2, single]
+  no-trailing-spaces: 2
+  comma-spacing: [2, { before: false, after: true }]
+  eqeqeq: [2, always]
+  linebreak-style: [2, unix]
+  semi: [2, always]
+  no-console: 0
+  no-inner-declarations: 0
+  no-constant-condition: 0
+  no-regex-spaces: 0
+
+extends: eslint:recommended
```

Comment:
Isn't the easy thing to just directly copy node's file? I've had horrendous times in the past when eslint bumps majors trying to update, and they used to bump majors or introduce incompatibilities pretty often. From what I saw, I'm not the only one. If its easy, I would just copy the file from nodejs, if that is more complex than it seems (perhaps it relies on some custom rules?), then I've no objection to this. Any eslint is better than none.